src/systems/fem_context.C

```diff
@@ -1532,6 +1554,7 @@ void FEMContext::pre_fe_reinit(const System &sys, const Elem *e)
   if (sys.use_fixed_solution)
     this->get_elem_fixed_solution().resize(n_dofs);
   this->get_elem_solution_rate().resize(n_dofs);
+  this->get_elem_solution_accel().resize(n_dofs);
```

Comment:
Agreed. Only thing that comes to mind is to force `FEMSystem` in the constructor and extract the time_solver.